### PR TITLE
Bugfix/spark tags dependency

### PR DESCRIPTION
**Pull request by pgruetter**
### What changes are included in the pull request?
Add spark-tags dependency for sdl-deltalake

### Why are the changes needed?
Somehow, sdl-deltalake doesn't always resolve the transitive dependency for spark-tags. When doing a fresh clone, maven would build correctly the first time. But when compiling a second time, I hit the error described here: https://stackoverflow.com/questions/42327777/spark-sql-errors. 

> error: missing or invalid dependency detected while loading class file 'package.class'. Could not access term annotation in package org.apache.spark, because it (or its dependencies) are missing.

### CODE REVIEW

File: sdl-deltalake/pom.xml [hunk 1]
```diff
@@ -67,6 +67,10 @@
 			<groupId>org.apache.spark</groupId>
 			<artifactId>spark-catalyst_${scala.minor.version}</artifactId>
 		</dependency>
+		<dependency>
```
Comment (zzeekk) [inline sdl-deltalake/pom.xml:70]: Scope=compile should be enough. And please add a comment maybe even referencing the stackoverflow issue.

**Review by zzeekk — COMMENTED** [review]
I had a similar issue in other projects and applied the same solution.

File: sdl-deltalake/pom.xml (hunk 1 above)
Comment (pgruetter) [reply sdl-deltalake/pom.xml:74]: Thanks, done in https://github.com/smart-data-lake/smart-data-lake/pull/360/commits/70668e7caf04a8106101e92638c749d3e87c410c.

Will wait with the merge until jms fix is finalized.

**Comment by pgruetter** [comment]
@zzeekk Discussed this one already, but I need one review ;-)

**Review by zzeekk — APPROVED** [review]

**Pull request merged**